libsolidity/interface/CompilerStack.h

```diff
@@ -186,7 +192,8 @@ class CompilerStack: boost::noncopyable
 	bool parseAndAnalyze();
 
 	/// Compiles the source units that were previously added and parsed.
-	/// @returns false on error.
+        /// @returns true if we should continue: there no
+	/// errors or there were errors but we want error recovery.
```

Comment:
I don't quite understand the comment :-). But it's probably not true, is it? It will actually throw if there were errors from which we recovered right now, won't it?

Reply:
This comment I added and is no probably no longer relevant since it was to document the behavior before the last set of changes. Therefore at this stage it doesn't make sense to clarify it if the the current before is simple `@returns false  on error`